### PR TITLE
Fix proxy rules test

### PR DESCRIPTION
**Pull request by josemigallas**
🥒

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ default: all
 
 ## This image is private and cannot be accessed by another third party than redhat.com employees
 ## You will need to build your own image as instructed in https://github.com/oracle/docker-images/tree/master/OracleDatabase/SingleInstance
-ORACLE_DB_IMAGE := quay.io/3scale/oracle:12.2.0.1-ee
+ORACLE_DB_IMAGE := quay.io/3scale/oracle:19.3.0-ee-ci-prebuilt
```
Comment (Martouta) [inline Makefile:32]: This does not belong to this PR 😄

Comment (josemigallas) [reply Makefile]: The base branch is not up to date with master 😓

File: features/step_definitions/proxy_steps.rb [hunk 2]
```diff
@@ -63,12 +63,12 @@ def stub_deploy_calls!
...
 end
 
-Given(/^I add a new mapping rule with method "([^"]*)" pattern "([^"]*)" delta "([^"]*)" and metric "([^"]*)"$/) do |method, pattern, delta, metric|
+Given(/^I add a new mapping rule with method "([^"]*)" pattern "([^"]*)" position "([^"]*)" and metric "([^"]*)"$/) do |method, pattern, delta, metric|
   click_on 'Add Mapping Rule'
   within(page.find('form.proxy_rule')) do
     find("select#proxy_rule_http_method option[value='#{method}']").select_option
     find('input#proxy_rule_pattern').set pattern
-    find('input#proxy_rule_delta').set delta
+    find('input#proxy_rule_position').set delta
```
Comment (Martouta) [inline features/step_definitions/proxy_steps.rb:71]:
```suggestion
    find('input#proxy_rule_position').set position
```
I guess 😄

**Comment by Martouta** [comment]
@josemigallas @didierofrivia check https://github.com/3scale/porta/pull/2274#discussion_r537618015
